src/scss/_admonitions.scss

```diff
@@ -1,8 +1,8 @@
 // Admonitions CSS originally inspired by https://squidfunk.github.io/mkdocs-material/getting-started/
 
-.admonition {
+div.admonition, .admonition{
   margin: 1.5625em auto;
```

Comment:
This bit now overrides the parent theme. Previously this margin was ignored. I am happy to undo this style change if necessary.